fontforge/splinechar.c

```diff
@@ -754,10 +752,6 @@ void SplinePointRound(SplinePoint *sp,real factor) {
 	sp->next->to->prevcp = sp->nextcp;
     if ( sp->prev!=NULL && sp->prev->order2 )
 	sp->prev->from->nextcp = sp->prevcp;
-    if ( sp->nextcp.x==sp->me.x && sp->nextcp.y==sp->me.y )
```

Comment:
Here I'm relying on the caller to also call SplineRefigure because otherwise the rounding won't take effect in the expanded coefficients. Both CVMouseUpShape and SplineSetsRound2Int do that on my reading.